### PR TITLE
Update design of Checkout page

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Update design for Checkout page to match Figma

## Justify why they are needed

I've excluded the cart summary -- will add that in a follow up PR.

## Jira issue(s): [GRW-1927]

[GRW-1927]: https://hedvig.atlassian.net/browse/GRW-1927?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1081** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1081" target="_blank">Graphite</a> 
    * **PR #1082** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1082" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1082?utm_source=stack-comment).

File: apps/store/src/components/CheckoutPage/CheckoutPage.tsx [hunk 1]
```diff
@@ -1,152 +1,111 @@
...
-                defaultValue={prefilledData.phoneNumber ?? undefined}
-              />
-            </Section>
+      <Section y={1}>
+        <SpaceBetween>
+          <SpaceFlex space={0.5} align="center">
+            <StepIcon />
+            <Text size="l">Dina kontaktuppgifter</Text>
+          </SpaceFlex>
+          <Text size="l">Hjälp</Text>
```
Comment (alebedev) [inline apps/store/src/components/CheckoutPage/CheckoutPage.tsx:41]: Localization?

File: apps/store/src/components/CheckoutPage/CheckoutPage.tsx [hunk 2]
```diff
@@ -1,152 +1,111 @@
...
-              <Text size="m">
-                After completing the purchase, you&apos;ll be able to connect payment.
-              </Text>
-              <SubmitButton loading={loading} signingStatus={props.signingStatus} />
-              {userError && <Text size="m">{userError.message}</Text>}
-            </Space>
+        <Space y={0.25}>
+          <PersonalNumberField
+            name={FormElement.PersonalNumber}
+            label="ÅÅÅÅMMDDXXXX"
```
Comment (alebedev) [inline apps/store/src/components/CheckoutPage/CheckoutPage.tsx:47]: We should have a key somewhere in purchase form configs already

File: apps/store/src/pages/checkout/index.tsx [hunk 3]
```diff
@@ -65,7 +66,7 @@ export const getServerSideProps: GetServerSideProps<NextPageProps> = async (cont
     const apolloClient = initializeApollo({ req, res })
     const [shopSession, translations] = await Promise.all([
       getCurrentShopSessionServerSide({ apolloClient, locale, req, res }),
-      serverSideTranslations(locale),
+      serverSideTranslations(locale, [I18nNamespace.Common, I18nNamespace.Checkout]),
```
Comment (alebedev) [inline apps/store/src/pages/checkout/index.tsx:69]: Do you think this optimization is already needed?

**Review by alebedev — APPROVED** [review]
Looks good, some nit comments and questions

File: apps/store/src/pages/checkout/index.tsx (hunk 3 above)
Comment (robinandeer) [reply apps/store/src/pages/checkout/index.tsx:69]: Hmmm what do you mean? We need to fetch the keys that are used on the page so we can server render properly, no?

**Comment by robinandeer** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1082?utm_source=stack-comment).


Job Reference: lDtBZ13Q9hNNbPBvf6yJ

**Pull request merged**